catalog/dags/providers/provider_api_scripts/smk.py

```diff
@@ -140,8 +140,7 @@ def _get_metadata(item: dict) -> dict:
         return meta_data
 
     def get_record_data(self, data: dict) -> dict | list[dict] | None:
-        license_info = get_license_info(license_url=data.get("rights"))
```

Comment:
This check was actually incorrect because `get_license_info` would never return `None`. For invalid licenses, it was returning a named tuple of `None`s.